src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,198 @@
+.pf-c-tree-view {
+  --pf-c-tree-view--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view--PaddingBottom: var(--pf-global--spacer--sm);
```

Comment:
Should this have top/bottom padding? If it's placed in a dropdown menu, for example, that padding will come from `.pf-c-dropdown__menu`